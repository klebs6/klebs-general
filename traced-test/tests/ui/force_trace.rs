use traced_test::traced_test;

#[traced_test(trace = true)]
fn EXPECT_TRACE_DISPLAYED_test_always_trace() {
    info!("This log should always be displayed.");
    assert!(true);
}
