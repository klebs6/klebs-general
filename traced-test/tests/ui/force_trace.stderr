error: custom attribute panicked
 --> tests/ui/force_trace.rs:3:1
  |
3 | #[traced_test(trace = true)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: not yet implemented: force trace on/off attribute still must be implemented

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/force_trace.rs:7:2
  |
7 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/force_trace.rs`
